TechnicoBackend/Repositories/UserRepository.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.EntityFrameworkCore;
+using TechnicoBackend.Data;
+using TechnicoBackend.Models;
+
+namespace TechnicoBackend.Repositories
+{
+    public class UserRepository
```

Comment:
Consider using an interface IUserRepository from which the current class would be inherited.